Lib/ufoLib/filenames.py

```diff
@@ -18,7 +18,7 @@ class NameTranslationError(Exception):
 	pass
 
 
-def userNameToFileName(userName, existing=[], prefix="", suffix=""):
+def userNameToFileName(userName, existing=(), prefix="", suffix=""):
```

Comment:
does it make a difference in terms of speed? I mean, using a mutable object as default value for a keyword argument may be questionable, but not sure it's gonna make much difference for speed

Reply:
mainly it's to signify that existing won't be mutated in the funciton.